GameServer/Logic/GameObjects/Spell.cs

```diff
@@ -205,18 +205,17 @@ public Spell(Champion owner, string spellName, byte slot)
             _targetType =
                 (float)Math.Floor(RAFManager.GetFloatValue(data, "Values", "SpellData", "TargettingType") + 0.5f);
 
-            JObject missileData;
-            if (!RAFManager.ReadMissileData(spellName, out missileData))
+            if (!RAFManager.ReadSpellData(spellName + "Missile", out data))
```

Comment:
It seems you've moved it here. Any reading related operation should really stay within the RAF manager, so I'd vote this to be moved back there.